src/pages/dashboard/nodeSignIn/NodeSignInPTForm.tsx

```diff
@@ -83,8 +73,8 @@ const NodeSignInPTForm = ({ goBack, signInData }: Props) => {
         })
         .finally(() => setLoading(false));
 
-      setIsPartnerPSPChecked(isPartnerPSP);
-      setIsPartnerCIChecked(isPartnerCI);
+      setIsPartnerPSPChecked(orgInfo.types.isPspBroker);
```

Comment:
Why are we using two different variables to give this check its value? 
Line 38 we use `orgIsPspDirect` and in line 76 we use `orgInfo.types.isPspBroker`
Same with the other check, line 39 `orgIsEcDirect` and line 77 `orgInfo.types.isEcBroker`